jsettlers.common/src/main/java/jsettlers/common/music/MusicPlayer.java

```diff
@@ -0,0 +1,113 @@
+package jsettlers.common.music;
+
+import java.io.File;
+import java.io.IOException;
+
+import javax.sound.sampled.AudioFormat;
+import javax.sound.sampled.AudioInputStream;
+import javax.sound.sampled.AudioSystem;
+import javax.sound.sampled.BooleanControl;
+import javax.sound.sampled.DataLine.Info;
+import javax.sound.sampled.FloatControl;
+import javax.sound.sampled.LineUnavailableException;
+import javax.sound.sampled.SourceDataLine;
+import javax.sound.sampled.UnsupportedAudioFileException;
+
+import static javax.sound.sampled.AudioSystem.getAudioInputStream;
+import static javax.sound.sampled.AudioFormat.Encoding.PCM_SIGNED;
+
+/* dependencies for .ogg and .mp3 files:
+ * ogg libraries: vorbisspi1.0.2.jar, tritonus_share.jar, jorbis-0.0.15.jar, jogg-0.0.7.jar
+ * mp3 libraries: jl1.0.1.jar, tritonus_share.jar, mp3spi1.9.5.jar
+ *
+ * gradle takes care of dependencies
+ */
+
+/**
+ *
+ * Supports .mp3 and .ogg music file streams
+ *
+ * dependencies for .ogg and .mp3 files: ogg libraries: vorbisspi1.0.2.jar, tritonus_share.jar, jorbis-0.0.15.jar, jogg-0.0.7.jar mp3 libraries: jl1.0.1.jar, tritonus_share.jar, mp3spi1.9.5.jar
+ *
+ * gradle takes care of dependencies
+ *
+ * @author MarviMarv
+ */
+public final class MusicPlayer {
```

Comment:
This class is basically a duplicate of SwingSoundPlayer except that the latter takes offsets into s3 sound file and this one takes ordinary sound files.

You might aswell merge the two.